test/functional/spec/page-spec.js

```diff
@@ -0,0 +1,74 @@
+/*global nemo:true, describe:true, it:true */
+var Nemo = require('nemo');
+var nemo = {};
+var util = require('../util');
+
+describe('@page@', function () {
+  before(function (done) {
+    nemo = Nemo(done);
+  });
+  after(function (done) {
+    nemo.driver.quit().then(done);
+  });
+  it('should execute high level functionality using generic methods', function (done) {
```

Comment:
Does this test name make sense? Are there non-generic methods? If so, there should be an example with those as well.